pkg/plugin/driver.go

```diff
@@ -116,10 +116,9 @@ func (h *Clickhouse) Converters() []sqlutil.Converter {
 // Macros returns list of macro functions convert the macros of raw query
 func (h *Clickhouse) Macros() sqlds.Macros {
 	return map[string]sqlds.MacroFunc{
-		"fromTime":   macros.FromTimeFilter,
-		"toTime":     macros.ToTimeFilter,
-		"timeFilter": macros.TimeFilter,
-		"table":      macros.Table,
-		"column":     macros.Column,
+		"fromTime":        macros.FromTimeFilter,
+		"toTime":          macros.ToTimeFilter,
+		"timeFilterMilli": macros.TimeFilterMilli,
```

Comment:
suggest "timeFilter_ms" to align with global var convention:  https://grafana.com/docs/grafana/latest/variables/variable-types/global-variables/#__interval_ms